client/src/components/StoreList.tsx

```diff
@@ -117,18 +123,73 @@ function StoreList() {
     }
   };
 
+  // Scan Store's QR code
+  const storeQR = async () => {
+    if (window.location == window.parent.location) {
+      const img = document.getElementById(
+        "sampleStoreQRImgSrc"
+      ) as HTMLImageElement;
+      processImageBarcode(img);
+    } else {
+      const imgReq = {
+        allowedMimeTypes: ["image/jpeg"],
+        allowedSources: ["camera"], // Restrict to camera scanning only
+      };
+      const imgRes = await window.microapps
+        .requestMedia(imgReq)
+        .then((res: any) => res);
+      setUploadImg(imgRes);
+    }
+  };
+
+  const processImageBarcode = async (img: HTMLImageElement) => {
+    const codeReader = new BrowserMultiFormatReader();
+    if (img != null) {
+      const result = await codeReader
+        .decodeFromImage(img)
+        .then((res: any) => res)
+        .catch((err: any) => {
+          console.log(err);
```

Comment:
remove console.log. These are alright for dev but better not to check in.

Reply:
resolved in #22